website/docusaurus.config.js

```diff
@@ -82,6 +82,15 @@ const config = {
   // - force trailing slashes for deploy previews
   // - avoid trailing slashes in prod
   trailingSlash: isDeployPreview,
+  headTags: [
+    {
+      tagName: "link", 
+      attributes: {
+        rel: "icon", 
+        href: "img/docusaurus.png"
+      }
+    },
+  ],
```

Comment:
```suggestion
```

To remove otherwise we have 2 icons on our website

![CleanShot 2022-10-12 at 17 27 14@2x](https://user-images.githubusercontent.com/749374/195384513-a7844132-eca2-433c-936d-4bc4f96d1cbf.png)

Note yours is relative so if I'm not on the home the image it resolved differently

- absolute: https://deploy-preview-8151--docusaurus-2.netlify.app/img/docusaurus.png => good
- relative: https://deploy-preview-8151--docusaurus-2.netlify.app/docs/api/img/docusaurus.png => bad